### PR TITLE
Update README*.md: Windows 10 and Other Unix-like Systems

### PR DESCRIPTION
**Pull request by jayr0d**
For Windows 10, Microsoft Visual C++ 14.0 or greater is needed for multidict.  Account for running scripts being disabled.

For Other Unix-like Systems, made formatting consistent with other sections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,7 +112,12 @@ pip install dali-rp2
 ### Installation on Other Unix-like Systems
 * install python 3.8 or greater
 * install pip3
-* `pip install dali-rp2`
+
+Then install DaLI:
```
Comment (macanudo527) [inline README.md:116]:
```suggestion
Then install DaLI-RP2:
```

**Review by macanudo527 — APPROVED** [review]
Just a nitpick. I haven't had this issue myself running on Win 11, but it could help others.

File: README.md (hunk 1 above)
Comment (jayr0d) [reply README.md:116]: There are other areas of README.md that say "Then install DaLI".  Would you like me to change them all?

Comment (eprbell) [reply README.md:116]: DaLI is the name of the software project, whereas dali-rp2 is the name of the Python package. I think in this context saying "Then install DaLI" is OK and shouldn't be too confusing. It's an English sentence directing the user to install the software, followed by the actual command to use in the CLI: I think the English sentence should use "DaLI" (which is less formal) and the command should (of course) use dali-rp2. Just my 2c.

Comment (macanudo527) [reply README.md:116]: Keep as is then. Thanks for the fixes!

**Pull request merged**